adm/Dockerfile-installed-bionic

```diff
@@ -18,45 +18,45 @@
 FROM ubuntu:bionic as adm-builder
 
 RUN apt-get update \
- && apt-get install gnupg -y
+  && apt-get install gnupg -y
 
 ENV VERSION=AUTO_STRICT
 
 RUN echo "deb http://repo.sawtooth.me/ubuntu/nightly bionic universe" >> /etc/apt/sources.list \
- && (apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 44FC67F19B2466EA \
- || apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 44FC67F19B2466EA) \
- && apt-get update \
- && apt-get install -y -q \
-    git \
-    libssl-dev \
-    openssl \
-    libzmq3-dev \
-    pkg-config \
-    python3 \
-    python3-cachetools \
-    python3-dev \
-    python3-grpcio \
-    python3-grpcio-tools \
-    python3-protobuf \
-    python3-pyformance \
-    python3-sawtooth-sdk \
-    python3-secp256k1 \
-    python3-stdeb \
-    python3-toml \
-    unzip
+  && (apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 44FC67F19B2466EA \
+  || apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 44FC67F19B2466EA) \
+  && apt-get update \
+  && apt-get install -y -q \
+  git \
+  libssl-dev \
+  openssl \
+  libzmq3-dev \
+  pkg-config \
+  python3 \
+  python3-cachetools \
```

Comment:
If we are going here can we get this in via pip? Or use a base image that has aarch64 versions of these packages

Reply:
Which package is causing you trouble?  These are all ubuntu upstream so they are all available in arm64 already.  

I suspect it is actually the python3-sawtooth-sdk package.

In any case I don't want to deal with ARM stuff here, since what I want to do is streamline the whole build.  There's still too much duplication of work, and seeing as how it will need to shift from bionic to the next LTS fairly soon, I'd like to deal with the complications introduced the last time that sort of thing was done.  Then we can deal with ARM.

Reply:
I also prefer the ubuntu packaged python3-* versions which have an extra layer of curation   And I have the opposite problem with the `diskcache` package which is not available in ubuntu:bionic as a deb, and has made a bit of a mess on its own.

Reply:
The pip was a bit of a nightmare on my arm hack, as you also get issues with EC libraries + the C build

Reply:
And all sorts is messing on the original base image, libzmq etc. I think anything with a C dep

Reply:
And yes agreed, the nasties bit of making it work with arm is the duplication